### PR TITLE
[WIP] Viz Download

### PR DESCRIPTION
**Pull request by phillipahereza**
## Description

Save viz when download button is clicked

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
download_image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by phillipahereza** [comment]
@kilemensi how did you fix the issue of Travis trying to use packages we don't have in Takwimu?

**Comment by kilemensi** [comment]
> @kilemensi how did you fix the issue of Travis trying to use packages we don't have in Takwimu?

For me it was [this issue](https://github.com/travis-ci/travis-ci/issues/7940#issuecomment-310759657) @phillipahereza

File: .travis.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ services:
 install:
 
   - pip install -r requirements.txt
+  - pip install requests==2.19.1
```
Comment (DavidLemayian) [inline .travis.yml]:
```suggestion
  - pip install requests==2.19.1
```

Not sure if this is needed or should be elsewhere.

**Comment by kilemensi** [comment]
This approach doesn't work for hosted viz due to CORS. We should double down on using HURUmap moving forward.

**Pull request closed without merging**